### PR TITLE
Chinese Translation

### PR DESCRIPTION
**Pull request by Harinlen**
A pure Chinese translation pull request. Merged the latest commit from origin.

### CODE REVIEW

File: game/tl/simplified_chinese/r3_end.rpy [hunk 1]
```diff
@@ -0,0 +1,1037 @@
...
+translate simplified_chinese end_3_236ec950:
+
+    # nvl_narrator "“By the way, that safe has a 4 digit passcode right?”"
+    nvl_narrator "　顺便一提，那个保险箱是有四位密码吧。"
+
+# game/r3_end.rpy:201
+translate simplified_chinese end_3_aa9c918b:
+
+    # nvl_narrator "“{b}How many cards are there in a suit in a deck of cards? The answer appears in the passcode.{/b}”"
+    nvl_narrator "　{b}「扑克牌中每种花色各有几张牌？答案会出现在四位密码中。」{/b}{/b}"
```
Comment (hananashi) [inline game/tl/simplified_chinese/r3_end.rpy:461]: two closing {/b}

File: game/tl/simplified_chinese/intro.rpy [hunk 2]
```diff
@@ -0,0 +1,622 @@
...
+translate simplified_chinese start_0e96fbb6:
+
+    # "“You are very ambitious when supporting others. You stop at nothing to help those around you reach their goals.”"
+    "『你是一位能热心帮助他人的人！你会为他人目标的实现而倾尽全力…！』"
+
+# game/intro.rpy:207
+translate simplified_chinese start_b49418a7:
+
+    # "“You are a very ambitious person. No matter what it is, you always want to give it 100%%, no, 200%% effort!”"
+    "『你是一位拥有野心的人。你会用100%，不，甚至用200%的精力去挑战一切事物！』"
```
Comment (hananashi) [inline game/tl/simplified_chinese/intro.rpy:428]: `%` needs to be escaped (as `%%`)

File: game/tl/simplified_chinese/r1_7_11.rpy [hunk 3]
```diff
@@ -0,0 +1,318 @@
...
+
+    # "We took two [conbini_food!t]s and went to pay for them."
+    "拿上两袋[conbini_food!t]，去店员那里结账。"
+
+# game/r1_7_11.rpy:54
+translate simplified_chinese r1_7_11_44e3cde3:
+
+    # "After buying them, we found empty seats in the eat-in corner."
+    "结好账后，又顺利地在店内的就餐区找到了空位。"
+    "{color=[p.who_args[color]]}[p]{/color}＆{color=[s.who_args[color]]}[s]{/color}" "「「いっただきまーす！」」"
```
Comment (hananashi) [inline game/tl/simplified_chinese/r1_7_11.rpy:60]: untranslated

File: game/tl/simplified_chinese/r1_7_11.rpy [hunk 4]
```diff
@@ -0,0 +1,318 @@
...
+translate simplified_chinese r1_7_11_c6862c26:
+
+    # s "The sauce is amazing!"
+    s "「酱汁也太好吃了吧！」"
+
+# game/r1_7_11.rpy:86
+translate simplified_chinese r1_7_11_2b639721:
+
+    # p "Right? This is supposed to be 100%% beef."
+    p "「是吧是吧！这个说是用的100%纯牛肉呢！」"
```
Comment (hananashi) [inline game/tl/simplified_chinese/r1_7_11.rpy:126]: % -> %%

File: game/tl/simplified_chinese/r2_village.rpy [hunk 5]
```diff
@@ -0,0 +1,906 @@
...
+translate simplified_chinese village_house_3_3_6df8d477:
+
+    # e "Yeah we got places to go, let’s go."
+    e "「说得对，我们有要去的地方，那么就出发吧」"
+
+# game/r2_village.rpy:449
+translate simplified_chinese village_house_4_1_374e488e:
+
+    # e "This a very dangerous domain; let’s explore somewhere else first!"
+    e "「此処は非常に危険な領域である、先ずは他の場所に訪れよう！」"
```
Comment (hananashi) [inline game/tl/simplified_chinese/r2_village.rpy:776]: untranslated

File: game/tl/japanese/r3_bomb.rpy [hunk 6]
```diff
@@ -368,7 +368,7 @@ translate japanese bomb_fail_fa5738e3:
 translate japanese bomb_fail_c98675fc:
 
     # "I rushed out of my room in a hurry to meet senpai."
-    pass
+    p "I rushed out of my room in a hurry to meet senpai."
```
Comment (hananashi) [inline game/tl/japanese/r3_bomb.rpy:371]: Extra change to JP script

**Review by hananashi — CHANGES_REQUESTED** [review]
Just a few small things.

I noticed your last commit removed the Chinese language option in the menu/the Chinese versions of images -- is there a reason for this?

**Comment by Harinlen** [comment]
> Just a few small things.
> 
> I noticed your last commit removed the Chinese language option in the menu/the Chinese versions of images -- is there a reason for this?

I was told that we are only required to submit the translation files and leave all the other codes to be the original repo. Should I submit all those codes?

**Comment by hananashi** [comment]
It would help a bit, but it's not hard to revert the last commit either; was just wondering if something had changed there. If you can get the two untranslated lines fixed I think it's good.

File: game/tl/simplified_chinese/r1_7_11.rpy (hunk 3 above)
Comment (gokoururi-git) [reply game/tl/simplified_chinese/r1_7_11.rpy:60]: translation：我开动了！

File: game/tl/simplified_chinese/r2_village.rpy (hunk 5 above)
Comment (gokoururi-git) [reply game/tl/simplified_chinese/r2_village.rpy:776]: translation：这个地方非常危险，我们先探索其他地方吧！

File: game/tl/japanese/r3_bomb.rpy (hunk 6 above)
Comment (gokoururi-git) [reply game/tl/japanese/r3_bomb.rpy:371]: The Japanese translation file we generated at the beginning is like this. If translation is required, it seems like that here should be:
```diff
    # "I rushed out of my room in a hurry to meet senpai."
-     p "I rushed out of my room in a hurry to meet senpai."
+     "人鱼飞奔出家门跑去和前辈见面。"
```

Comment (hananashi) [reply game/tl/japanese/r3_bomb.rpy:371]: Ah I'll just remove it; it's untranslated because it's combined with the line after.

Comment (gokoururi-git) [reply game/tl/japanese/r3_bomb.rpy:371]: okay，thanks a lot!

**Pull request merged**